pkg/foxtrot/db_test.go

```diff
@@ -0,0 +1,74 @@
+package foxtrot
+
+import (
+	"context"
+	"errors"
+	"io/ioutil"
+	"os"
+	"testing"
+
+	_ "github.com/mattn/go-sqlite3"
+	"github.com/stretchr/testify/require"
+)
+
+func TestCreateGetUser(t *testing.T) {
+	db, err := newDB("")
+	require.NoError(t, err)
+	defer db.close()
+
+	u := &User{Name: "alice", passwordHash: "###"}
+	err = db.createUser(context.Background(), u)
+	require.NoError(t, err)
+
+	u2, err := db.getUser(context.Background(), "alice")
```

Comment:
where the vars have no name :musical_note: (u2, get it?)

Reply:
totally. I love [the streets](https://music.youtube.com/watch?v=WpfoHIoii6A&feature=share).

Reply:
I will listen to that later